ci/homebrew-tap/ci-deploy.sh

```diff
@@ -29,4 +30,10 @@ if [[ $do_push == true ]]; then
   git push origin master
 else
   echo "Refraining from push since it's a dry run"
+  sed -i "s/pypi.io/$pypi_host/" git-machete.rb
+  # install git-machete from local formula with homebrew
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" < /dev/null
+  echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"' >> /home/circleci/.bash_profile
+  eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```

Comment:
Hmmm is it needed to both add the `eval` to bash_profile AND to run it in the current shell?

Reply:
Also, I'd avoid hardcoding HOME path, just use `~`

Reply:
yes, `eval` to bash profile is no necessary  in this case indeed. 

HOME path is hardcoded already brew's ruby install script, here is explanation from their installation site:
The installation script installs Homebrew to /home/linuxbrew/.linuxbrew using sudo if possible and in your home directory at ~/.linuxbrew otherwise. Homebrew does not use sudo after installation. Using /home/linuxbrew/.linuxbrew allows the use of more binary packages (bottles) than installing in your personal home directory.

The prefix /home/linuxbrew/.linuxbrew was chosen so that users without admin access can ask an admin to create a linuxbrew role account and still benefit from precompiled binaries.